frontend/src/components/camp/CollaboratorListItem.vue

```diff
@@ -4,8 +4,8 @@
       <v-img src="https://i.pravatar.cc/300" />
     </v-list-item-avatar>
     <v-list-item-content>
-      <v-list-item-title>{{ collaborator.user().displayName }}</v-list-item-title>
-      <v-list-item-subtitle>{{ collaborator.user().mail }}</v-list-item-subtitle>
+      <v-list-item-title>{{ collaborator.user ? collaborator.user().displayName : "" }}</v-list-item-title>
+      <v-list-item-subtitle>{{ collaborator.inviteEmail }}</v-list-item-subtitle>
```

Comment:
This will continue displaying the `inviteEmail` even after I've accepted the invitation with an account with a different email.

Reply:
No, the inviteEmail is deleted when an campCollaboration switches to the status established.